netbench/netbench/src/scenario/id.rs

```diff
@@ -0,0 +1,42 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use core::fmt::Write;
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Debug, Default, PartialEq, Serialize, Deserialize, Hash)]
+pub struct Id(String);
+
+impl Id {
+    pub(crate) fn hasher() -> Hasher {
+        Hasher::default()
+    }
+}
+
+#[derive(Debug, Default)]
+pub struct Hasher {
+    hash: sha2::Sha256,
```

Comment:
What was the reasoning behind using this versus something like https://crates.io/crates/uuid?

Reply:
I wanted the scenario id to be deterministic based on the contents so when you execute a scenario between 2 peers they can agree on the the scenario to use.